Rakefile

```diff
@@ -79,4 +80,1837 @@ task :update_measures do
+      walls_values = get_hpxml_file_walls_values(hpxml_file, walls_values)
+      windows_values = get_hpxml_file_windows_values(hpxml_file, windows_values)
+      doors_values = get_hpxml_file_doors_values(hpxml_file, doors_values)
+      heating_systems_values = get_hpxml_file_heating_systems_values(hpxml_file, heating_systems_values)
+      cooling_systems_values = get_hpxml_file_cooling_systems_values(hpxml_file, cooling_systems_values)
+      heat_pumps_values = get_hpxml_file_heat_pumps_values(hpxml_file, heat_pumps_values)
+      hvac_controls_values = get_hpxml_file_hvac_control_values(hpxml_file, hvac_controls_values)
+      hvac_distributions_values = get_hpxml_file_hvac_distribution_values(hpxml_file, hvac_distributions_values)
+      duct_leakage_measurements_values = get_hpxml_file_duct_leakage_measurements_values(hpxml_file, duct_leakage_measurements_values)
+      ducts_values = get_hpxml_file_ducts_values(hpxml_file, ducts_values)
+      ventilation_fans_values = get_hpxml_file_ventilation_fan_values(hpxml_file, ventilation_fans_values)
+      water_heating_systems_values = get_hpxml_file_water_heating_system_values(hpxml_file, water_heating_systems_values)
+      hot_water_distributions_values = get_hpxml_file_hot_water_distribution_values(hpxml_file, hot_water_distributions_values)
+      water_fixtures_values = get_hpxml_file_water_fixtures_values(hpxml_file, water_fixtures_values)
+      clothes_washers_values = get_hpxml_file_clothes_washer_values(hpxml_file, clothes_washers_values)
+      clothes_dryers_values = get_hpxml_file_clothes_dryer_values(hpxml_file, clothes_dryers_values)
+      dishwashers_values = get_hpxml_file_dishwasher_values(hpxml_file, dishwashers_values)
+      refrigerators_values = get_hpxml_file_refrigerator_values(hpxml_file, refrigerators_values)
+      cooking_ranges_values = get_hpxml_file_cooking_range_values(hpxml_file, cooking_ranges_values)
+      ovens_values = get_hpxml_file_oven_values(hpxml_file, ovens_values)
+      lightings_values = get_hpxml_file_lighting_values(hpxml_file, lightings_values)
+      plug_loads_values = get_hpxml_file_plug_load_values(hpxml_file, plug_loads_values)
+      misc_loads_schedules_values = get_hpxml_file_misc_loads_schedule_values(hpxml_file, misc_loads_schedules_values)
+    end
+
+    hpxml_doc = HPXML.create_hpxml(**hpxml_values)
+    hpxml = hpxml_doc.elements["HPXML"]
+
+    if File.exists? File.join(tests_dir, derivative)
+      old_hpxml_doc = XMLHelper.parse_file(File.join(tests_dir, derivative))
+      created_date_and_time = HPXML.get_hpxml_values(hpxml: old_hpxml_doc.elements["HPXML"])[:created_date_and_time]
+      hpxml.elements["XMLTransactionHeaderInformation/CreatedDateAndTime"].text = created_date_and_time
+    end
+
+    HPXML.add_site(hpxml: hpxml, **site_values) unless site_values.nil?
+    building_occupancys_values.each do |building_occupancy_values|
+      HPXML.add_building_occupancy(hpxml: hpxml, **building_occupancy_values)
+    end
```

Comment:
Hmm, why are we looping here? There can only be a single occupancy value.

Reply:
I'm going to make a couple small changes so I'll change this.

Reply:
Should probably change here, too? https://github.com/NREL/OpenStudio-HPXML/blob/master/Rakefile#L409-L411